http/api/transport.go

```diff
@@ -76,14 +77,32 @@ func decodeRequest(_ context.Context, r *http.Request) (interface{}, error) {
 	return msg, nil
 }
 
-// TODO: contact an auth provider
 func authorize(r *http.Request) (string, error) {
 	var apiKey string
+	var id string
+
 	if apiKey = r.Header.Get("Authorization"); apiKey == "" {
 		return "", errUnauthorizedAccess
 	}
 
-	return apiKey, nil
+	identityURL := os.Getenv("HTTP_ADAPTER_MANAGER_URL") + "/identity"
+	req, err := http.NewRequest("POST", managerURL, nil)
+	if err != nil {
+		return "", err
+	}
+
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("Authorization", apiKey)
+
+	c := &http.Client{}
+	resp, err := c.Do(req)
+	if err != nil {
+		return "", err
```

Comment:
Make sure this error is properly handled. The safest way is to utilize domain errors.

Reply:
What do you mean by this? I thought that upper layer above `transport.go` handles this error, so I just propagate it.

Reply:
Safely ignore this one, it was deprecated by #138 .